src/LinAlg/hiopMatrixRajaDense.cpp

```diff
@@ -1385,6 +1385,9 @@ void hiopMatrixRajaDense::row_max_abs_value(hiopVector &ret_vec)
 {  
   assert(ret_vec.get_size() == m());
   ret_vec.setToZero();
+  if(0 == m()) {
+    return;
+  }  
```

Comment:
```suggestion
  if(0 == m_local_) {
    return;
  }  
```
I know this won't change anything because HiOp matrices are not partitioned by rows, but I would make this check explicitly with respect to _local_ row numbers.